src/components/Context/injectHighEntropyUserAgentHints.js

```diff
@@ -24,12 +24,18 @@ export default navigator => {
   return (xdm, logger) => {
     try {
       return navigator.userAgentData
-        .getHighEntropyValues(highEntropyUserAgentHints)
+        .getHighEntropyValues(highEntropyUserAgentHints.map(hint => hint[0]))
         .then(hints => {
           const userAgentClientHints = {};
           highEntropyUserAgentHints.forEach(hint => {
-            if (Object.prototype.hasOwnProperty.call(hints, hint)) {
-              userAgentClientHints[hint] = hints[hint];
+            const hintName = hint[0];
+            const hintType = hint[1];
```

Comment:
Style nitpick that you can ignore:

You can replace 
```js
highEntropyUserAgentHints.forEach(hint => {
  const hintName = hint[0];
  const hintType = hint[1];
  // …
});
```
with 
```js
highEntropyUserAgentHints.forEach(([hintName, hintType]) => {
  // …
});
```

Reply:
I like that.